examples/README.md

```diff
@@ -0,0 +1,125 @@
+# go-qemu examples
+
+This folder contains example applications that demonstrate the use of the go-qemu and go-qemu/hypervisor packages.
+
+If the program is executed using the **"unix"** named network (i.e locally from the hypervisor host), the user under which the program executes on needs belong to 'libvirtd' group
+so the account has access to **'/var/run/libvirt/libvirt-sock'**
+
+In case of executing the program remotely and connecting thru **"tcp"**, you could configure libvirtd on the hypervisor host to allow tcp connections. Of course, this is not secured.
+
+To see the list of posible arguments for each program, use **--help**
+
+#### hypervisor_domain_list
+
+[hypervisor_domain_list](./hypervisor_domain_list) demonstrates how to use the [hypervisor](../hypervisor) package to obtain a list of the domains from the connected hypervisor. The list of domains returned is of type: [go-qemu/Domain](../domain.go#L37).
```

Comment:
Chances are pretty good that the line number of the definition of `Domain` will change in the future.

Let's link to the godoc instead.